### DISCUSSION TITLE
[Suggestions] It will great to make more achievement badges 🔰

### DISCUSSION THREAD

**Discussion by ranggakd** [community/community#33510] [Other Feature Feedback, Questions, & Ideas]
I think it will rewarding when GitHub could add more badges for more inclusive significant milestones and activities that are newbie-friendly and encourage people to contribute better. Just like [dev.to](https://dev.to/swissdevjobs/the-complete-list-of-30-dev-to-badges-and-how-to-catch-them-all-almost-4d3o) does

**Comment by ranggakd** [comment]
This is a badge gamification GitHub proposal to encourage participation and contribution 🙃

**Reply by secxndary** [reply]
true, true. i agree with this statement.

**Reply by ranggakd** [reply]
You should upvote then, so we could unify people with the same request 😁
